src/lib/components/backoffice/Sidebar/Sidebar.js

```diff
@@ -40,6 +40,8 @@ class Sidebar extends Component {
     const checkInActive = activePath.includes(BackOfficeRoutes.checkIn);
     const checkOutActive = activePath.includes(BackOfficeRoutes.checkOut);
 
+    const query = '&sort=mostrecent&p=1';
```

Comment:
the issue is that it might be that we have some searches that do not support the `mostrecent` sorting. Do we really need this?